python/pyspark/ml/feature.py

```diff
@@ -2064,6 +2112,20 @@ def getNumBuckets(self):
         """
         return self.getOrDefault(self.numBuckets)
 
+    @since("3.0.0")
+    def setNumBucketsArray(self, value):
+        """
+        Sets the value of :py:attr:`numBucketsArray`.
+        """
+        return self._set(numBucketsArray=value)
+
+    @since("3.0.0")
+    def getNumBucketsArray(self):
+        """
+        Gets the value of numBucketsArray or its default value.
```

Comment:
:py:attr:`numBucketsArray`?

Reply:
@viirya Thanks for your comments. 
I looked setters and getters in other places, for some reason, the doc of all the setters have ```:py:attr:```, but the majority of the getters doc don't have ```:py:attr:```. I guess I will not add ```:py:attr:``` for this getter because all the other getters doc in `QuantileDiscretizer` don't have ```:py:attr:```

Reply:
oh, I see. Then it is fine. :)